README.md

```diff
@@ -1,2 +1,49 @@
 # Leaderboard
-In this project I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I will develop a first working version of the app following a wireframe, but without styling - just focus on functionality
+
+In this project i acheived the following
+
```

Comment:
- [ ] [OPTIONAL] I would highly recommend you to add a screenshot of your project to make your Readme look even better!